package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "point-of-view",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
can you please remove this? Bumping the version is responsibility of the releaser.

Reply:
ok sure, sorry ...